index.html

```diff
@@ -217,22 +213,22 @@ <h2 class="skill-name">Skills</h2>
 			</li>
 		</ul>
 	</section>
-	<section  class="contact-container" id="footer-container-id">
+	<section  class="contact-container">
 		<div class="contact-form">
 			<h2 class="contact-me">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>		
 			<form action="https://formspree.io/f/xknagbpo" method="post">
 				<ul class="contact-elements">
-					<li class="text-input">
-						<input type="text" name="fullname"maxlength="30" placeholder="Full name" required>
+					<li class="fullname-input">
+						<input class="fullname-input" type="text" name="fullname"maxlength="30" placeholder="Full name" required>
 					</li>
 					<li>
-						<input type="email" name="email" placeholder="Email Address" required>
+						<input class="email-input" type="email" name="email" placeholder="Email Address" required>
 					</li>
 					<li>
-						<textarea  type="text" name="message" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10" required></textarea>
+						<textarea class="textarea-input" type="text" name="message" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10" required></textarea>
 					</li>
 					<li>
-						<button type="submit">Get in touch</button>
+						<button class="form-submit-btn" type="submit">Get in touch</button>
 					</li>
```

Comment:
- [ ] Kindly make sure that all your button have the interactive style (hover , active) to improve the user experience :call_me_hand: 
 
| ![Screenshot from 2023-01-20 15-08-30](https://user-images.githubusercontent.com/80458199/213702368-bd8354d3-6690-40b3-85be-913e89d2015f.png) |  ![Screenshot from 2023-01-20 15-08-15](https://user-images.githubusercontent.com/80458199/213702414-73b8ef3c-4ee3-4e0e-b37f-aee581eef3a8.png)  |  ![Screenshot from 2023-01-20 15-07-57](https://user-images.githubusercontent.com/80458199/213702457-49fa5da8-28f9-4bdd-a157-14cf964ce41b.png)|


### What you could do 

```bash
button {
   cursor: pointer;
}

button:hover {
    color: xxx;
    background-color: yyyy;
}
````